src/sentry/models/release.py

```diff
@@ -65,9 +65,9 @@ class Release(Model):
     class Meta:
         app_label = 'sentry'
         db_table = 'sentry_release'
-        unique_together = (('project_id', 'version'),)
+        unique_together = (('organization', 'version'),)
```

Comment:
We should do this in two separate migrations.

First, add the UNIQUE for `organization, version`, then DROP `project, version` as a follow up. Otherwise trying to do these two at the same time is going to be a mess.

And we can add the new index `CONCURRENTLY` with some manual SQL like we've done elsewhere.